rewrite-xml/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ dependencies {
     api("com.fasterxml.jackson.core:jackson-annotations")
 
     compileOnly(project(":rewrite-test"))
+    compileOnly("org.slf4j:slf4j-api:1.7.+")
```

Comment:
We don't often log from our recipes, as it's hard to uniformly present that output in the various ways recipes might run. Instead we often opt to add markers, which we might then in turn log, or present in some other way to the user, as appropriate. If at all possible I'd opt for markers, or no logging statements here too. What are your thoughts on that?

Reply:
Logging is simpler, requires less knowledge of the intrinsics of OR.  But I'm interested in learning a different way to do this type of communication / user feedback. Is there any good example from where I can learn?

Reply:
Showing how markers are used. Agree with @timtebeek we cannot have logging statements in core recipes: https://app.moderne.io/recipes/org.openrewrite.java.search.FindMethods?organizationId=T3BlblJld3JpdGU%3D&defaults=W3sibmFtZSI6Im1ldGhvZFBhdHRlcm4iLCJ2YWx1ZSI6Im9yZy5vcGVucmV3cml0ZS5tYXJrZXIuU2VhcmNoUmVzdWx0IGZvdW5kKC4uKSJ9LHsibmFtZSI6Im1hdGNoT3ZlcnJpZGVzIiwidmFsdWUiOiJUcnVlIn1d

Reply:
Your example uses DataTables which are not available on OpenRecipe yet, just on Moderne.io.

Reply:
Resolving this discussion as the logging was removed, and data tables are now available:
https://docs.openrewrite.org/running-recipes/data-tables